applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -216,6 +228,94 @@ load_conference(DocId, Context) ->
 %%--------------------------------------------------------------------
 %% @private
 %% @doc
+%% Load details of the participants of a conference
+%% @end
+%%--------------------------------------------------------------------
+-spec load_conference_details(cb_context:context(), path_token()) -> cb_context:context().
+load_conference_details(Context, ConfId) ->
+    AccountRealm = wh_util:get_account_realm(cb_context:account_id(Context)),
+    Req = [{<<"Realm">>, AccountRealm}
+           ,{<<"Fields">>, []}
+           ,{<<"Conference-ID">>, ConfId}
+           | wh_api:default_headers(?APP_NAME, ?APP_VERSION)
+          ],
+    ReqResp = whapps_util:amqp_pool_collect(Req
+                                            ,fun wapi_conference:publish_search_req/1
+                                            ,{'ecallmgr', 'true'}
+                                           ),
+    case ReqResp of
+        {'error', _} -> cb_context:add_system_error('conference_search_error', Context);
```

Comment:
@danielfinke 
How would you feel about a more typical reply such as 
https://github.com/2600hz/kazoo/blob/master/applications/crossbar/src/cb_context.erl#L1018
or just the not_found?